test/Microsoft.ML.OnnxTransformTest/Microsoft.ML.OnnxTransformTest.csproj

```diff
@@ -5,10 +5,32 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.PipelineInference\Microsoft.ML.PipelineInference.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.DnnImageFeaturizer.ResNet18\Microsoft.ML.DnnImageFeaturizer.ResNet18.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Predictor.Tests\Microsoft.ML.Predictor.Tests.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.ML.Onnx.TestModels" Version="0.0.2-test" />
   </ItemGroup>
+
+  <ItemGroup>
+    <DnnPrepModelFiles Include="$(RepoRoot)bin\obj\DnnImageModels\ResNetPrepOnnx\ResNetPreprocess.onnx" />
```

Comment:
Don't use `bin\obj` in an MSBuild path. If you really mean `bin\obj`, you should use `$(ObjDir)`. But typically we define new properties for common folder paths instead of duplicating the paths everywhere.

Given my other comment about these files getting downloaded to `$(PackageAssetsPath)$(MSBuildProjectName)`, these should be updated to:

```xml
<DnnPrepModelFiles Include="$(PackageAssetsPath)Microsoft.ML.DnnImageFeaturizer.ResNet18\contentFiles\any\any\DnnImageModels\ResNetPreprocess.onnx" />
```